integration/testdata/ca_cert_app/README.md

```diff
@@ -0,0 +1,16 @@
+### Ruby client auth server
+
+Usage:
+
+```
+pack build appimage -b gcr.io/paketo-buildpacks/ca-certificates -b gcr.io/paketo-buildpacks/ruby
```

Comment:
I don't think you need to explicitly include the CA certificates buildpack here since it's in the Ruby buildpack order groups.

Reply:
Yeah, I think we just did that for testing purposes. I can change it.